### PR TITLE
openshift-ansible: add jobs to test Gluster deploy on AWS

### PR DESCRIPTION
**Pull request by vrutkovs**
Added custom jobs for release-3.1x jobs to test gluster deployment using AWS
provisioner

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml [hunk 1]
```diff
@@ -264,6 +264,70 @@ presubmits:
...
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-gluster-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-gluster
+          name: job-definition
+          subPath: cluster-launch-e2e-openshift-ansible.yaml
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Please use the ci-operator config to generate these

/cc @bbguimaraes

Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: `ci-operator-prowgen` would generate jobs for `prow-job-cluster-launch-e2e`, but this one needs `prow-job-cluster-launch-e2e-openshift-ansible`.

Once the new test option is added to ci-operator I'll update operator config to have these generated

Comment (stevekuznetsov) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: _You_ should be adding new options there :)

Comment (vrutkovs) [reply ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-release-3.10-presubmits.yaml]: Created https://github.com/openshift/ci-operator/issues/175 to track this

**Comment by vrutkovs** [comment]
/close

This would be reworked using ci-operator-prowgenerator

**Pull request closed without merging**